linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/rule.yml

```diff
@@ -11,26 +11,19 @@ description: |-
 
     The DoD required text is either:
     <br /><br />
-    <tt>You are accessing a U.S. Government (USG) Information System (IS) that
-    is provided for USG-authorized use only. By using this IS (which includes
-    any device attached to this IS), you consent to the following conditions:
-    <br />-The USG routinely intercepts and monitors communications on this IS
-    for purposes including, but not limited to, penetration testing, COMSEC
-    monitoring, network operations and defense, personnel misconduct (PM), law
-    enforcement (LE), and counterintelligence (CI) investigations.
-    <br />-At any time, the USG may inspect and seize data stored on this IS.
-    <br />-Communications using, or data stored on, this IS are not private,
-    are subject to routine monitoring, interception, and search, and may be
-    disclosed or used for any USG-authorized purpose.
-    <br />-This IS includes security measures (e.g., authentication and access
-    controls) to protect USG interests -- not for your personal benefit or
-    privacy.
-    <br />-Notwithstanding the above, using this IS does not constitute consent
-    to PM, LE or CI investigative searching or monitoring of the content of
-    privileged communications, or work product, related to personal
-    representation or services by attorneys, psychotherapists, or clergy, and
-    their assistants. Such communications and work product are private and
-    confidential. See User Agreement for details.</tt>
+    <tt>You are accessing a U.S. Government (USG) Information System (IS) that is provided for USG-authorized use only.<br />
+    <br />
+    By using this IS (which includes any device attached to this IS), you consent to the following conditions:<br />
+    <br />
+    -The USG routinely intercepts and monitors communications on this IS for purposes including, but not limited to, penetration testing, COMSEC monitoring, network operations and defense, personnel misconduct (PM), law enforcement (LE), and counterintelligence (CI) investigations.<br />
+    <br />
+    -At any time, the USG may inspect and seize data stored on this IS.<br />
+    <br />
+    -Communications using, or data stored on, this IS are not private, are subject to routine monitoring, interception, and search, and may be disclosed or used for any USG-authorized purpose.<br />
+    <br />
+    -This IS includes security measures (e.g., authentication and access controls) to protect USG interests--not for your personal benefit or privacy.<br />
```

Comment:
This was the part which incorrectly had spaces around the two dashes (old line 26).